libclamav/scanners.c

```diff
@@ -4591,6 +4591,11 @@ cl_error_t cli_magic_scan(cli_ctx *ctx, cli_file_t type)
                 ret = cli_scanegg(ctx);
             break;
 
+        case CL_TYPE_ONENOTE:
+            if (SCAN_PARSE_ARCHIVE && (DCONF_ARCH & ARCH_CONF_EGG))
```

Comment:
I found an issue. This should be ARCH_CONF_ONENOTE (which doesn't exist, have to edit dconf.h to add it).

Reply:
Fixed.